packages/api/index.js

```diff
@@ -1,34 +1,31 @@
-let bundle = require('./app/bundle.api.js');
-let injected = require('./injected');
-let methods = require('./methods');
-let utils = require('./utils');
+import fetch from 'node-fetch';
+
+import * as bundle from './app/bundle.api';
+import * as injected from './injected';
+
 let actualApp;
+export const internal = bundle.lib;
 
-async function init(config = {}) {
+export * as methods from './methods';
+export * as utils from './utils';
+
+export async function init(config = {}) {
   if (actualApp) {
     return;
   }
 
-  global.fetch = require('node-fetch');
+  global.fetch = fetch;
 
   await bundle.init(config);
   actualApp = bundle.lib;
 
-  injected.send = bundle.lib.send;
+  injected.override(bundle.lib.send);
```

Comment:
This is the only API change I had to do, as ESM exports are read only, so had to provide a function to achieve the same functionality.